client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -36,7 +36,6 @@ function ReaderSubscriptionListItem( {
 	const siteIcon = get( site, 'icon.img' );
 	const feedIcon = get( feed, 'image' );
 	const streamUrl = getStreamUrl( feedId, siteId );
-	const siteUrl = url || getSiteUrl( { feed, site } );
```

Comment:
a subscription should always have a feed ID and a URL. it may or may not have a blog ID / site ID